common_sensor_launch/launch/velodyne_node_container.launch.py

```diff
@@ -90,12 +90,24 @@ def create_parameter_dict(*args):
     cropbox_parameters["negative"] = True
 
     vehicle_info = get_vehicle_info(context)
-    cropbox_parameters["min_x"] = vehicle_info["min_longitudinal_offset"]
-    cropbox_parameters["max_x"] = vehicle_info["max_longitudinal_offset"]
-    cropbox_parameters["min_y"] = vehicle_info["min_lateral_offset"]
-    cropbox_parameters["max_y"] = vehicle_info["max_lateral_offset"]
-    cropbox_parameters["min_z"] = vehicle_info["min_height_offset"]
-    cropbox_parameters["max_z"] = vehicle_info["max_height_offset"]
+    cropbox_parameters["min_x"] = vehicle_info["min_longitudinal_offset"] - LaunchConfiguration(
```

Comment:
I remember you have to evaluate the LaunchConfiguration values using `context` in order to do four arithmetic operations in launch files.

Reply:
Thank you! :+1: 
https://github.com/tier4/aip_launcher/pull/25#discussion_r812244110

Reply:
Here is an example.
https://github.com/tier4/autoware_launch/blob/927534386a16a6a38679173cb7ccc884fe7c74cb/perception_launch/launch/obstacle_segmentation/ground_segmentation/ground_segmentation.launch.py#L50

Reply:
[52cf47c](https://github.com/tier4/aip_launcher/pull/25/commits/52cf47c0b04f68db9d2e93cbc86c094a4844ed9e)